examples/websocket-client-cookie/websocket_client_cookie.bal

```diff
@@ -0,0 +1,76 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+
+// The HTTP client configurations associated with [enabling cookies](https://ballerina.io/swan-lake/learn/api-docs/ballerina/http/records/CookieConfig.html).
+http:ClientConfiguration clientEPConfig = {
+    cookieConfig: {
+        enabled: true
+    }
+};
+
+public function main() {
+    // Create a new HTTP client by giving the URL and the client configuration.
+    http:Client httpClient = new("http://localhost:9095/cookie-demo", clientEPConfig);
+
+    // Initialize an HTTP request.
+    http:Request request = new;
+
+    // Send a username and a password as a JSON payload to the backend.
+    json jsonPart = {
+        name: "John",
+        password: "p@ssw0rd"
+    };
+    request.setJsonPayload(jsonPart);
+
+    // Send an outbound request to the `login` backend resource.
+    var loginResp = httpClient->post("/login", request);
+
+    if (loginResp is http:Response) {
+            // This response contains the cookies added by the backend server.
+            // Get the login response message.
+            string|error loginMessage = loginResp.getTextPayload();
```

Comment:
Why do this payload access is needed?

Reply:
If the server accepts the login request, it will set the string [payload](https://github.com/ballerina-platform/ballerina-distribution/pull/668/files#diff-40b6aad65e7d45ea424e4fa2e5e8358bR43) in the response. So here, we check whether getting that string in the payload or not